waspc/data/Generator/templates/server/src/routes/apis/index.js

```diff
@@ -0,0 +1,25 @@
+{{={= =}=}}
```

Comment:
I would suggest we rewrite this file to Typescript to get the benefits of Typescript :)

Reply:
Rewritten!